install

```diff
@@ -1,2 +1,8 @@
 #! /bin/bash
 python3 -m pip install .
+if [ ! -d "$HOME/.config/sootty/save" ] 
+then
+    mkdir -p $HOME/.config/sootty/save
+else
+    echo Found save files for sootty
```

Comment:
Add quotes: `echo "Found save files for sootty"`

Reply:
Will do, but is there any reason?

Reply:
Bash can get a little screwy sometimes, so I think its better to be explicit when using strings.